draft-ietf-tls-tls13.md

```diff
@@ -1415,7 +1426,9 @@ Note:
   low-entropy sources is not secure. A low-entropy secret, or password, is
   subject to dictionary attacks based on the PSK binder.  The specified PSK
   authentication is not a strong password-based authenticated key exchange even
-  when used with Diffie-Hellman key establishment.
+  when used with Diffie-Hellman key establishment.  That is, it does not provide
+  resistance againt an attacker that can observe the handshake performing
```

Comment:
s/handshake performing/handshake and perform/ ?

Reply:
I'm going with a slightly different disambiguation.